Worker.Dockerfile

```diff
@@ -38,4 +38,4 @@ RUN npm run build:scanner_worker
 # Ensure scancode is accessible from the PATH
 ENV PATH="/venv/bin:${PATH}"
 
-CMD cd apps/scanner_worker && npm run start
\ No newline at end of file
+CMD cd apps/scanner_worker && npm run start
```

Comment:
Still, there is no change in this file except for adding the newline, so it shouldn't be part of this commit.